README.md

```diff
@@ -50,7 +50,9 @@ happily rely on it for `Span` propagation. To start a new (blocking child)
         ...
         span, ctx := opentracing.StartSpanFromContext(ctx, "operation_name")
         defer span.Finish()
-        span.LogEvent("xyz_called")
+        span.LogFields(
+            log.String("event", "xyz called"),
+            log.Int("request.size", request.Size()))
```

Comment:
The idea here is that `"xyz called"` happened at some specific moment, and the call had a certain `"request.size"`... whether that "call" should be its own Span is a tradeoff, but I do think there's an implied timestamp for this event.